x/ccv/child/types/params.go

```diff
@@ -16,15 +22,26 @@ func ParamKeyTable() paramtypes.KeyTable {
 }
 
 // NewParams creates new child parameters with provided arguments
-func NewParams(enabled bool) Params {
+func NewParams(enabled bool, blocksPerDistributionTransmission int64,
+	distributionTransmissionChannel, providerFeePoolAddrStr, consumerRedistributeFrac string) Params {
 	return Params{
-		Enabled: enabled,
+		Enabled:                           enabled,
+		BlocksPerDistributionTransmission: blocksPerDistributionTransmission,
+		DistributionTransmissionChannel:   distributionTransmissionChannel,
+		ProviderFeePoolAddrStr:            providerFeePoolAddrStr,
+		ConsumerRedistributeFrac:          consumerRedistributeFrac,
 	}
 }
 
 // DefaultParams is the default params for the child module
 func DefaultParams() Params {
-	return NewParams(false)
+	return NewParams(
+		false,
+		1000, // about 2 hr at 7.6 seconds per blocks
+		"",
+		"",
+		"0.5",
```

Comment:
I'd set the default to 0 (all rewards go to the provider) since it's the simplest case. The alternative entails the consumer to figure out a mechanism to distribute its share of the rewards.

Reply:
Sounds good. Done